docs/CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Infrastructure targets can now mount secret vars to a job regardless of environment variables.
+  Secret variables are passed as a separate argument, they're no longer mixed with regular env vars.
+  Infrastructure plugins have to be updated accordingly due to interface change.
+  Specifically, `deploy_job` function of `JobDeployer` class now has
+  a new argument `runtime_secret_vars: Dict[str, str]` with secret env vars for a job.
```

Comment:
Can you imagine a way to make this change non-breaking? So that all plugins don't have to be updated instantly once a new version rolls in (which is operationally difficult), but lazily when there's time?

Reply:
Yes, one way to do that would be to inspect a method signature and check if it has this argument defined. 

I know :) it's not ideal codewise, but it avoids much bigger operational problem (at least I think it will be problem, as there are dozens of plugins - how can you re-upload all right after starting newer RT version?). When all plugins are updated, such checking code can be removed - so ideally, even few days later. Last but not least, Python is good in meta inspections of code.

Reply:
Alright, you convinced me. I'll use inspection to make plugins backward-compatible.

Reply:
Take a look at https://github.com/TheRacetrack/racetrack/pull/395/commits/9c2e0bced05a5b20bfbff6345962ddd691b55830 . It wasn't that bad.